Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Use the official Ubuntu base image
+FROM ubuntu:latest
+
+# Set environment variables
+ENV TZ=Europe/Zurich
+
+# Install necessary dependencies
+RUN apt-get update && apt-get install -y \
+    wget \
+    bzip2 \
+    ca-certificates \
+    curl \
+    git \
+    # Install tzdata to set the timezone, otherwise timing tests of probtest will fail with
+    # ValueError: time data 'Sun Jun 26 20:11:23 CEST 2022' does not match format '%a %b %d %H:%M:%S %Z %Y'
+    tzdata \
+    && apt-get clean
+
+
+COPY . /probtest
+RUN cd /probtest && ./setup_miniconda.sh -p /opt/conda
+# Add conda to PATH
+ENV PATH=/opt/conda/miniconda/bin:$PATH
+
+# only unpinned env works on aarch64
```

Comment:
Do you know why the pineed version is not working? did you try to update it? In principal it would be prefered to use the pinned here, as this one is tested.

Reply:
There was a version constraint that could not be met.
Could be that for aarch64 not the full set of available packages it there.

Yes I agree. Installing the pinned version is faster.

This is something we can sort out later, for now this is sufficient.
But before the container is used in ICON officially we should revise.
I open an issue.

Reply:
See #57

Reply:
Ok, I thought by just updating the pinned environment this might go away (then for the moment pinned and unpinned should be the same). But as we are not deploying often it is kind of pined, it would be just nice that deploying it will not have side effect on when deployed.